src/Orleans.Runtime.Abstractions/Streams/SiloPersistentStreamConfigurator.cs

```diff
@@ -8,9 +8,35 @@
 using System;
 using System.Collections.Generic;
 using System.Text;
+using Orleans.Storage;
 
 namespace Orleans.Streams
 {
+    public class PersistentStreamStorageConfigurationValidator : IConfigurationValidator
+    {
+        private IServiceProvider services;
+        private const string pubsubStoreName = "PubSubStore";
+        private string streamProviderName;
+        public PersistentStreamStorageConfigurationValidator(IServiceProvider services, string streamProviderName)
```

Comment:
It makes sense to have a PersistentStreamStorageConfigurationValidator, but I'm wondering if this specific logic would be better as a per pubsub system validator, rather than a single validator expected to know about all the possible pubsub systems.

Reply:
if pubsub system is injectable, then I think we should have a per pubsub validator. But that is not the case yet. So, I suggest let's worry about that when we have other pubsub systems in the future.